src/utils/urls.js

```diff
@@ -144,6 +144,10 @@ function getRoomUrl(roomId) {
   return concatParts(roomsPrefix, encodeURIComponent(roomId));
 }
 
+function getRoomDetailsUrl(roomId) {
+  return concatParts(roomsPrefix, roomId);
+}
```

Comment:
I renamed this to `getRoomUrl` (the function above) :laughing: (too late, I know).